src/execution/execute.ts

```diff
@@ -832,6 +867,14 @@ function completeValue(
   result: unknown,
   asyncPayloadRecord?: AsyncPayloadRecord,
 ): PromiseOrValue<unknown> {
+  if (exeContext.signal?.isAborted) {
```

Comment:
Hmm, also: I think completeValue is probably the wrong place to put this, or at least if you do put this here you need to be a bit more careful. Because if `result` is an array of Promises, now you are "leaking" the Promises by not calling `catch` or putting them in a `Promise.all` or whatever, which seems incorrect... it's generally a bad idea to have unhandled Promises and can lead to weird logging and such.

I don't know the best place to put this check (and maybe it should go in multiple places) but putting it at the top of executeField (ie, *before* doing a resolver call, not midway through after) seems reasonable?

(Or you could leave it here but also be careful to find any `Promise`s in `result` and put an empty `catch` on them or something...)